src/rules/use_isnan.rs

```diff
@@ -54,7 +54,7 @@ impl Visit for UseIsNaNVisitor {
         if is_nan_identifier(&ident) {
           self.context.add_diagnostic(
             bin_expr.span,
-            "useIsNaN",
+            "use-is-na-n",
```

Comment:
Why is only this one different from eslint ?

Reply:
Fixed already